examples/qtensor/qtensor.morpho

```diff
@@ -1,17 +1,26 @@
+/* Problem of a 2D nematic confined to a disk with parallel anchoring boundary condition, modeled by a Q-tensor free energy.
+Showcases: AreaIntegral, LineIntegral, MeshRefiner, POVRaytracer, Graphics
+This script showcases a few good aspects of Morpho:
+* It uses AreaIntegral to define the polynomial form of free energy. 
+* It uses mesh refinement to get a better resolution for the solution at the nematic defect locations, where there is more distortion in the field value.
+* To visualize the Q-tensor, we need to see both the director (orientations) and the scalar order parameter (degree of order) together. This script shows how that can be done easily by just adding the graphics together.
+*/
 import meshtools
 import optimize
 import plot
 import povray
 
```

Comment:
// Define parameters that control the script options

Reply:
You mean this for the flags `refine_adaptively`, etc, right?

Reply:
yup, I think this was meant for line 12